### ISSUE TITLE
Should Agency GET endpoints return MDS version in their responses?

### ISSUE THREAD

**Issue by thekaveman** [openmobilityfoundation/mobility-data-specification#851]
Agency GET endpoints return their data payload, and optionally the JSON API pagination fields, e.g. for [Vehicles](https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.0.0/agency/README.md#vehicle---list):

```jsonc
{
    "vehicles": [ ... ],
    "links": {
        "first": "https://...",
        "last": "https://...",
        "prev": "https://...",
        "next": "https://..."
    }
}
```

Should the MDS `version` be included in this response? Like it is for Provider, Policy, Jurisdiction responses? 

```jsonc
{
    "version": "2.0.0",
    "vehicles": [ ... ],
    "links": {
        "first": "https://...",
        "last": "https://...",
        "prev": "https://...",
        "next": "https://..."
    }
}
```

**Comment by schnuerle** [comment]
Yes it should for consistency across the spec. I'll update the draft and let you know when done if you handle the schema.

**Comment by schnuerle** [comment]
Updated with this [commit](https://github.com/openmobilityfoundation/mobility-data-specification/pull/837/commits/334fc1e48c76fb336b5d2b9a6bdf145071942a8e). Please review then we can close this issue.

**Comment by thekaveman** [comment]
Those are the only spots that show/mention the payload structure so I think this is fine.

Just noting that `/stops/{stop_id}` also defines a GET under Agency (but the docs are lighter); I'll add `version` there too.

**Comment by thekaveman** [comment]
Transferring to OpenAPI repo since the work is done in MDS repo.
